server/mergin/auth/controller.py

```diff
@@ -545,15 +545,13 @@ def create_user():
 @auth_required(permissions=["admin"])
 def get_server_usage():
     data = {
-        "active_monthly_contributors": [
-            current_app.ws_handler.monthly_contributors_count(),
-            current_app.ws_handler.monthly_contributors_count(month_offset=1),
-            current_app.ws_handler.monthly_contributors_count(month_offset=2),
-            current_app.ws_handler.monthly_contributors_count(month_offset=3),
-        ],
-        "projects": Project.query.count(),
+        "active_monthly_contributors": current_app.ws_handler.monthly_contributors_count(),
+        "projects": Project.query.filter(Project.removed_at.is_(None)).count(),
         "storage": files_size(),
-        "users": User.query.count(),
+        "users": User.query.filter(
+            is_(User.username.ilike("deleted_%"), False) | User.active,
```

Comment:
I can be wrong.. :man_shrugging:  Try to deactivate a user in a test, please.

Reply:
This issue is caused by soft delete of things in previous months. We can remove some user from projects or something like this.

This filter is coming from codebase, probably we filter just removed projects.

Reply:
I would not mind to count only active users, the chance that they have inactive user which is not scheduled for removal (aka. banned user) is small and it would not affect statistics much

Reply:
I think we need to review how to filter removed users properly. I can take a look next week.